tasks/state-installed.yml

```diff
@@ -39,6 +39,11 @@
     - k3s_primary_control_node
     - k3s_server_manifests_templates | length > 0
 
+- import_tasks: build/preconfigure-k3s-static-pod-manifests.yml
+  when:
+    - k3s_control_node
```

Comment:
Does it make sense to make this `k3s_primary_control_node` ?

Reply:
It depends I guess? If you want to deploy Pods there (such as for kube-vip), I think the manifest should go on all of the control plane nodes.

Reply:
You are probably right, let's keep it this way. I know for the other manifest option it makes sense to put them on a single controller node. Static pods might behave differently.

Reply:
Static pods are directly tied to their kubelet (they essentially are a container outside of the cluster, represented by a mirror object inside the cluster) therefore they must be deployed individually to each control node.